mne/gui/_ieeg_locate_gui.py

```diff
@@ -0,0 +1,859 @@
+        self._fig, self._ax = plt.subplots(figsize=(width, height), dpi=dpi)
+        self._fig.subplots_adjust(bottom=0, left=0, right=1,
+                                  top=1, wspace=0, hspace=0)
+        self._ax.set_facecolor('k')
+        # clean up excess plot text, invert
+        self._ax.invert_yaxis()
+        self._ax.set_xticks([])
+        self._ax.set_yticks([])
+        plt.close(self._fig)  # close extra figure, gets embedded
+        super(SlicePlot, self).__init__(self._fig)
+
+
+class IntracranialElectrodeLocator(QMainWindow):
+    """Locate electrode contacts using a coregistered MRI and CT."""
+
+    def __init__(self, info, trans, aligned_ct, subject=None,
+                 subjects_dir=None, groups=None, verbose=None):
+        """GUI for locating intracranial electrodes.
+
+        .. note:: Images will be displayed using orientation information
+                  obtained from the image header. Images will be resampled to
+                  dimensions [256, 256, 256] for display.
+        """
+        # initialize QMainWindow class
+        super(IntracranialElectrodeLocator, self).__init__()
+
+        if not info.ch_names:
+            raise ValueError('No channels found in `info` to locate')
+
+        # store info for modification
+        self._info = info
+        self._verbose = verbose
+
+        # load imaging data
+        self._subject_dir = _check_subject_dir(subject, subjects_dir)
+        self._load_image_data(aligned_ct)
+
+        self._ch_alpha = 0.5
+        self._radius = int(_CH_PLOT_SIZE // 100)  # starting 1/200 of image
+        # initialize channel data
+        self._ch_index = 0
+        # load data, apply trans
+        self._head_mri_t = _get_trans(trans, 'head', 'mri')[0]
+        self._mri_head_t = invert_transform(self._head_mri_t)
+        # load channels, convert from m to mm
+        self._chs = {name: apply_trans(self._head_mri_t, ch['loc'][:3]) * 1000
+                     for name, ch in zip(info.ch_names, info['chs'])}
+        self._ch_names = list(self._chs.keys())
+        # set current position
+        if np.isnan(self._chs[self._ch_names[self._ch_index]]).any():
+            self._ras = np.array([0., 0., 0.])
+        else:
+            self._ras = self._chs[self._ch_names[self._ch_index]].copy()
+        self._current_slice = apply_trans(
+            self._ras_vox_t, self._ras).round().astype(int)
+        self._group_channels(groups)
+
+        # GUI design
+
+        # Main plots: make one plot for each view; sagittal, coronal, axial
+        plt_grid = QGridLayout()
+        self._plts = [SlicePlot(self), SlicePlot(self), SlicePlot(self)]
+        plt_grid.addWidget(self._plts[0], 0, 0)
+        plt_grid.addWidget(self._plts[1], 0, 1)
+        plt_grid.addWidget(self._plts[2], 1, 0)
+        self._renderer = create_3d_figure(
+            size=(400, 400), bgcolor='w', scene=False)
+        plt_grid.addWidget(self._renderer.figure.plotter)
```

Comment:
This has a failure in the `--pre` branch that uses latest GitHub `main`/`master` of PyVista and PyVistaQt, can you replicate and fix?